### DISCUSSION TITLE
Document Renovate Flux integration

### DISCUSSION THREAD

**Discussion by danports** [fluxcd/flux2#2394] [Proposals] [answered]
Hi all,

Flux is an awesome tool. I'm grateful for how it has automated a lot of my IT work. I recently discovered [Renovate](https://github.com/renovatebot/renovate/), another awesome tool that automatically opens PRs for dependencies that have been updated. The combination of Flux and Renovate is a match made in heaven:

1. Flux deploys the resources according to the source Git repo.
2. Renovate opens PRs on that repo whenever new versions of components are available (including Flux itself).
3. Flux upgrades the components when those PRs are merged.

What would be the best place to document this integration? I noticed:

1. https://fluxcd.io/integrations/
2. https://fluxcd.io/docs/installation/#upgrade (which describes automated upgrades via GitHub Actions, similar to what Renovate can do)

**Comment by dholbach** [comment, marked as answer]
1. Mentioning it under `/integrations` would be great!
1. And yes I personally feel `/docs/installation/#upgrade` could be a good start - not sure how others feel about where to best place the documentation of integrations?

@kingdonb Do you maybe have an opinion?

**Reply by kingdonb** [reply]
I do agree and I opened a specific issue for it here: https://github.com/fluxcd/website/issues/740

(I'll link it back to this discussion!)

**Reply by dholbach** [reply]
Thanks for filing the issue @kingdonb - I replied on it.

**Comment by dholbach** [comment]
1. This might also be a good use-case for https://github.com/fluxcd/website/issues/558 :-)

**Reply by dholbach** [reply]
https://github.com/fluxcd/website/pull/741 adds this section and links off to https://docs.renovatebot.com/modules/manager/flux/

**Reply by HaveFun83** [reply]
awesome :+1:
